test/citestutils.js

```diff
@@ -170,11 +189,11 @@ function testBackup(params, databaseName, outputStream, callback) {
   }
 
   // Wait for the promises and then assert
-  return Promise.all(promises)
+  return await Promise.all(promises)
```

Comment:
I don't think this should `await` since the promise chain is getting returned to the caller (and eventually to mocha which will handle any necessary waiting)
```suggestion
  return Promise.all(promises)
```